proposals/ecosystem/index.bs

```diff
@@ -1025,19 +1023,609 @@ The [=Application Registration=] must be associated with the
+                        &lt;cf2d40d3-858e-42f7-8b11-d021b5989ed2/#registration&gt ,
+                        &lt;628a7371-3a9b-49f9-a7aa-2e9c93e0150b/#registration&gt .
+  </pre>
+</figure>
+
+A [=Data Registration=] extends the
+[eco:Registration](ecosystem.ttl#Registration) class with the
+[eco:DataRegistration](ecosystem.ttl#DataRegistration) subclass.
+
+A [=Data Registration=] includes any number of [=Registered Data Instances=]
+via the
+[eco:hasRegisteredDataInstance](ecosystem.ttl#hasRegisteredDataInstance)
+property.
+
+The [eco:DataRegistration](ecosystem.ttl#DataRegistration) subclass
+extends the [eco:Registration](ecosystem.ttl#Registration) class with
+the following properties:
+
+* [eco:registeredShapeTree](ecosystem.ttl#registeredShapeTree) - URI of the
+    [=shape tree=] that is being registered.
+* [eco:hasRegisteredDataInstance](ecosystem.ttl#hasRegisteredDataInstance) -
+    Any number of [=Registered Data Instances=] of the
+    [=shape tree=] associated via
+    [eco:registeredShapeTree](ecosystem.ttl#registeredShapeTree).
+
+
+<figure id="fig-datareg-registration">
+  <figcaption>A [=Data Registration=] at
+  https://alice.pod.example/data/49a4057a-6ade-4a9d-9869-085a00925259/#registration
+  linking to several [=Registered Data Instances=]</figcaption>
+  <pre highlight="turtle" line-highlight="5" line-numbers="true">
+  <#registration>
+    a eco:DataRegistration ;
+    eco:registeredBy &lt;https://alice.pod.example/profile/id#me&gt; ;
+    eco:registeredWith &lt;https://app.example/id#agent&gt; ;
+    eco:registeredAt "2020-04-04T20:15:47.000Z"^^xsd:dateTime .
+    eco:registeredShapeTree &lt;https://tree.example/commonnote#notebook-tree&gt; ;
+    eco:hasRegisteredDataInstance &lt;3f56c32e-96c6-4b0c-b941-eadc2bf98b18/#notebook&gt; ,
+                                  &lt;16e1eae9-20a5-489d-8380-8c07ca3805c4/#notebook&gt; ,
+                                  &lt;886785d2-6c23-46ee-9e19-5ac36b7bcd71/#notebook&gt; ,
+                                  &lt;dae5015c-d5f9-4ede-b9eb-7ca7a19854a9/#notebook&gt; .
+    </pre>
+</figure>
+
+## Structure and Validation ## {#datareg-structure}
+
+The resources associated with an [=Agent's=] [=Data Registries=] conform
+to and extend the general patterns detailed in Registrar [[#reg-structure]].
+
+Consequently, this section focuses on the unique composition and validation of
+[=Data Registrations=] in a [=Data Registry=], and the
+[=Registered Data Instances=] contained in each [=Data Registration=].
+
+### Structure of Data Registrations in a Data Registry ### {#datareg-registrations-structure}
+
+<figure id="fig-datareg-registry-structure">
+  <figcaption>A [=Data Registry=] at
+  https://alice.pod.example/data/#registry</figcaption>
+  <img src="datareg-structure.svg" width=100%/>
+</figure>
+
+In the figure above, a [=Data Registry=] instance is stored in the RDF graph
+of the `/data` container at `https://alice.pod.example/data/#registry`. This
```

Comment:
suggest it be labeled as `/data/` (trailing slash) to be consistently name containers.